documentation/tut/toy-example-from-scratch.rst

```diff
@@ -181,7 +181,7 @@ Yes, that is quite a large battery :)
 We can also look at the battery asset in the UI of FlexMeasures (in Docker, the FlexMeasures web server already runs, on your PC you can start it with ``flexmeasures run``).
 Visit `http://localhost:5000/assets <http://localhost:5000/assets>`_ (username is "toy-user@flexmeasures.io", password is "toy-password") and select "toy-battery":
 
-.. image:: https://github.com/FlexMeasures/screenshots/raw/main/tut/toy-schedule/asset-view.png
+.. image:: https://github.com/FlexMeasures/screenshots/raw/main/tut/toy-schedule/asset-view-without-data.png
     :align: center
 
 .. note:: You won't see the map tiles, as we have not configured the :ref:`MAPBOX_ACCESS_TOKEN`. If you have one, you can configure it via ``flexmeasures.cfg`` (for Docker, see :ref:`docker_configuration`).
```

Comment:
This note can go, if there is no map visible.

I have a different idea here: Instead of an empty asset view, we can show a screenshot of the dashboard, showing the asset. Then this note can stay, of course.

Reply:
if we try to access to assets then it's better to show only the assets and not the dashboard.

Reply:
![image](https://github.com/FlexMeasures/flexmeasures/assets/59763365/15cb2bbe-d0a3-47af-a61d-98010248cd2a)